syntax = "proto3";

package bayes.bob;

// Data for seasonal jobbing for a given month.
message MonthlySeasonalJobbingStats {
  // The departements in which there are seasonal jobs.
  repeated MonthlySeasonalDepartementStats departement_stats = 1;
}

// Data about a seasonal departement for a given month.
message MonthlySeasonalDepartementStats {
  // The ID of the departement.
  string departement_id = 1;

  // The total number of seasonal offers in this departement for a given month.
  int32 departement_seasonal_offers = 2;

  // The 5 job groups which hire the most sorted by most to least.
  repeated JobGroupStats job_groups = 3;

  // Departement with location formulation, e.g.: "en Corr√®ze".
  string departement_in_name = 4;
}

message JobGroupStats {
  // The ROME ID of the job group.
  string rome_id = 1;

  // The ROME name of the job group.
  string name = 2;

  // The number of job offers over two years for this job group.
  int32 offers = 3;

  // The market score for this job group.
  int32 market_score = 4;
}
